openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -49,4 +47,4 @@ def handle(self, *args, **options):
             if not course_keys:
                 log.fatal('No courses specified.')
```

Comment:
Assuming log fatal does not have a side effect of raising an error, wouldn't we typically want to raise one with a similar message from the same location?